src/bp/core/services/module/resources-loader.ts

```diff
@@ -51,6 +62,44 @@ export class ModuleResourceLoader {
     await this._loadModuleResources()
   }
 
+  async enableResources() {
+    const ghost = this.ghost.global()
+
+    for (const path of this.globalPaths) {
+      const files = await ghost.directoryListing(path, undefined, undefined, true)
+      await Promise.all(
+        files
+          .filter(name => name.startsWith(DISABLED_PREFIX))
+          .map(file => ghost.renameFile(path, file, file.replace(DISABLED_PREFIX, '')))
+      )
+    }
+
+    const hooks = await ghost.directoryListing('/hooks', undefined, undefined, true)
+    await Promise.all(
+      hooks
+        .filter(f => this.hookMatcher.test(f) && path.basename(f).startsWith(DISABLED_PREFIX))
```

Comment:
that works but could have just split the normalized path by `/` and take the [1] index?

Reply:
Yep, but I found it easier to let the normalizing & extraction to basename